src/style.css

```diff
@@ -0,0 +1,30 @@
+html {
+  background-color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.outer {
+  margin: 20px 20px 40px;
+  width: 328px;
+  background-color: rgb(48, 141, 192);
+  border: 1px solid #e95d49;
+}
+
+.middle {
+  margin: 0;
+  background-color: rgb(255, 165, 0);
+  border: 1px solid #e95d49;
+}
+
+.inner {
+  margin: 0;
+  background-color: #ddd;
+  border: 1px solid #e95d49;
```

Comment:
you repeat border property three times, create common class or combine with padding 10px